app/models/apple/config.rb

```diff
@@ -61,19 +60,21 @@ def self.find_or_build_private_feed(podcast)
       )
     end
 
+    # this doesn't seem to be called anywhere?
+    # is this a helper for setup via console, with that stdin.get?
```

Comment:
Yep, this is used to onboard new shows

Eventually, we can retrofit this as a programmatic pathway for connecting feeds to delegated delivery.